packages/wdio-appium-service/tests/launcher.test.js

```diff
@@ -0,0 +1,177 @@
+import { AppiumLauncher } from '../src/launcher'
+import cp from 'child_process'
+import fs from 'fs-extra'
+import path from 'path'
+
+jest.mock('child_process', () => ({
+    spawn: jest.fn(),
+}))
+jest.mock('fs-extra', () => ({
```

Comment:
Can you move both of this into the mocks directory (`packages/wdio-appium-service/tests/__mocks__` which is the normal way to put these mocks into seperate files. Jest then automatically mocks them.

Reply:
When I move the two `jest.mock()` to separate files (`child_process.js` and `fs-extra.js` in a `__mocks__` folder in the root of `packages/wdio-appium-service`- not in the `tests` folder) and add a `import jest from 'jest'` I get the following error. I haven't used jest and babel before, so I have no idea how to fix this. I placed the `jest.mock()` in the `launcher.test.js` file to match how `wdio-selenium-standalone-service` does.

```
Test suite failed to run

    babel-plugin-jest-hoist: The module factory of `jest.mock()` is not allowed to reference any out-of-scope variables.
    Invalid variable access: jest
    Whitelisted objects: Array, ArrayBuffer, Boolean, DataView, Date, Error, EvalError, Float32Array, Float64Array, Function, Generator, GeneratorFunction, Infinity, Int16Array, Int32Array, Int8Array, InternalError, Intl, JSON, Map, Math, NaN, Number, Object, Promise, Proxy, RangeError, ReferenceError, Reflect, RegExp, Set, String, Symbol, SyntaxError, TypeError, URIError, Uint16Array, Uint32Array, Uint8Array, Uint8ClampedArray, WeakMap, WeakSet, arguments, console, expect, isNaN, jest, parseFloat, parseInt, require, undefined, DTRACE_NET_SERVER_CONNECTION, DTRACE_NET_STREAM_END, DTRACE_HTTP_SERVER_REQUEST, DTRACE_HTTP_SERVER_RESPONSE, DTRACE_HTTP_CLIENT_REQUEST, DTRACE_HTTP_CLIENT_RESPONSE, global, process, Buffer, clearImmediate, clearInterval, clearTimeout, setImmediate, setInterval, setTimeout.
    Note: This is a precaution to guard against uninitialized mock variables. If it is ensured that the mock is required lazily, variable names prefixed with `mock` (case insensitive) are permitted.

      at invariant (node_modules/babel-plugin-jest-hoist/build/index.js:15:11)
      at newFn (node_modules/@babel/traverse/lib/visitors.js:193:21)
      at NodePath._call (node_modules/@babel/traverse/lib/path/context.js:53:20)
      at NodePath.call (node_modules/@babel/traverse/lib/path/context.js:40:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:88:12)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:118:16)
      at TraversalContext.visitMultiple (node_modules/@babel/traverse/lib/context.js:85:17)
```

Reply:
Can you help me out, @christian-bromann?

Reply:
@MortenGregersen  How are you importing the modules into your `__mocks__` file?

Reply:
I didn't import the modules into a `__mocks__` file. I moved the `jest.mock()` code from the `launcher.test.js` to separate files (child_process.js and fs-extra.js in a `__mocks__` folder in the root of packages/wdio-appium-service- not in the tests folder) and added a `import jest from 'jest'`.

Reply:
I got it to work somehow... 🤔